sdks/python/apache_beam/internal/gcp/json_value.py

```diff
@@ -105,7 +105,9 @@ def to_json_value(obj, with_type=False):
     return extra_types.JsonValue(object_value=json_object)
   elif with_type:
     return to_json_value(get_typed_value_descriptor(obj), with_type=False)
-  elif isinstance(obj, (str, unicode)):
+  elif isinstance(obj, (str, unicode, bytes)):
```

Comment:
I'd suggest to make a separate branch for `bytes`:
```
elif isinstance(obj, bytes):
  return extra_types.JsonValue(string_value=obj.decode('utf-8'))
```
To keep the existing behavior on Python 2, we can place this branch after 
`  elif isinstance(obj, (str, unicode)):`   branch.
```

Reply:
sg. done